### PR TITLE
Adds macOS support for local environment

### PR DESCRIPTION
**Pull request by jwmatthews**
This PR adds support for running locally on macOS.

It is using the 'summit' tags and intended to run same functionality we had for summit demos locally on Mac.  I'd like this to be merged into master, then I'd like to get 'dev' to take these changes in.  

I'm in the process of testing this on Linux and EC-2.

Once merged this will address https://github.com/fusor/catasb/issues/42

### CODE REVIEW

**Comment by jwmatthews** [comment]
@cfchase @dymurray please take a look at this for Linux and EC-2.
Intent is to get this merged into master, then later get these changes into 'dev'

I've run tests on macOS, Linux, and EC-2.
Looking good so far.

**Comment by cfchase** [comment]
@jwmatthews tested on Linux for both local and ec2.  Both worked without issue.

File: local/mac/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+### Overview
+These playbooks will:
+  * Setup Origin through `oc cluster up`
+  * Install Service Catalog on Origin
+  * Install Ansible Service Broker on Origin
+
+### Pre-Reqs
+  * ```socat``` needs to be installed
+        brew install socat
```
Comment (cfchase) [inline local/mac/README.md]: I'm guessing this needed to be:
```bash
brew install socat
```

Comment (jwmatthews) [inline local/mac/README.md]: Thanks Chris, fixed.

**Comment by cfchase** [comment]
ACK

**Comment by dymurray** [comment]
ACK, success for me on EC2 and local environments for Linux. John mentioned that Andrew also had success on Mac so I am good to merge.

**Pull request merged**